src/atom1.ts

```diff
@@ -148,13 +148,6 @@ export default (ins: Feed) => {
       entry.published = item.published.toISOString();
     }
 
-    // source
-
-    // rights
-    if (item.copyright) {
-      entry.rights = item.copyright;
-    }
-
     base.feed.entry.push(entry);
```

Comment:
@decebal Why removing this?

Reply:
@jpmonette the `rights` are added on the feed, I tried to test it and couldn't find any support in official atom1 documentation for it being on each entry. I still kept the feed entry https://github.com/jpmonette/feed/blob/b73c0d19d795199c09b2325eb79e55fdf3e4c871/src/atom1.ts#L59 .